.github/workflows/test_gen.yaml

```diff
@@ -194,8 +194,11 @@ jobs:
       matrix:
         python-version:
         - 3.8.12
+        spark-version:
+        - 3.3.1
+        - 3.5.0
```

Comment:
Curious if there is a reason behind the workflows changes? They seem unrelated to the main change in the PR.

Reply:
Need to update the workflow here so that the tests won't fail due to linux image not get updated. I checked out the workflow from main branch to this release branch.